lib/music_album.rb

```diff
@@ -0,0 +1,18 @@
+require_relative './item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(publish_date, archived: false, on_spotify: false)
+    super(publish_date, archived: archived)
+    @on_spotify = on_spotify
+  end
+
+  private
+
+  def can_be_archieved?
+    return true if super && @on_spotify == true
+
+    false
+  end
```

Comment:
[Optional] you can write the function with `super && @on_spotify`  it will do the same thing as your implementation.

Reply:
@darikmohammed thanks for your feedback